website/payments/admin.py

```diff
@@ -68,6 +71,9 @@ class PaymentAdmin(admin.ModelAdmin):
         "topic",
         "notes",
         "batch",
+        "moneybird_invoice_id",
+        "moneybird_financial_statement_id",
+        "moneybird_financial_mutation_id",
```

Comment:
Don't have the payments app depend on the moneybird sync app. Instead, create a separate admin (possibly, you can extend the PaymentAdmin and reregister the payments admin in the moneybird app, though!, see https://stackoverflow.com/questions/25964028/how-do-i-register-a-model-that-is-already-registered-in-admin)